Backpack/src/main/java/net/skyscanner/backpack/calendar/view/BpkCalendarLabelsViewModel.kt

```diff
@@ -38,10 +43,22 @@ internal class BpkCalendarLabelsViewModel(
 
   private var labels: Map<LocalDate, BpkCalendarLabelViewModel> = emptyMap()
 
+  private val placeholder = BpkCalendarLabelViewModel(
+    label = CalendarLabel(
+      text = "-",
```

Comment:
This should be an n-dash instead of a hyphen as it's (sort of) copy. It looks the same in code as the font is monospaced, but should look better in the app

```suggestion
      text = "–",
```